pycon/test_settings.py

```diff
@@ -0,0 +1,8 @@
+from .dev_settings import *  # NOQA
+
```

Comment:
We might want to set `DEBUG=False` here explicitly - otherwise, some things such as exception handling, are going to be different between test and prod.